project/Build.scala

```diff
@@ -287,7 +287,8 @@ object Build extends sbt.Build {
       "org.webjars.bower" % "vis" % "4.7.0",
       "org.webjars.bower" % "clipboard.js" % "0.1.1",
       "org.webjars.npm" % "dashing-deps" % "0.0.8",
-      "org.webjars.npm" % "dashing" % "0.3.8"
+      "org.webjars.npm" % "dashing" % "0.3.8",
+      "com.softwaremill" %% "akka-http-session" % "0.1.4"
```

Comment:
Better put backend dependencies before any frontend dependencies (before the first occurrence of "org.webjar")